packages/cli/src/commands/governance/approve.ts

```diff
@@ -21,6 +21,9 @@ export default class Approve extends BaseCommand {
     const id = res.flags.proposalID
     this.kit.defaultAccount = account
     const governance = await this.kit.contracts.getGovernance()
+    const governanceApproverMultiSig = await this.kit.contracts.getMultiSig(
```

Comment:
Please consider making this command work both with and without a multisig, e.g. by adding a `--multisig` flag, or even adding an automatic check that looks up whether or not the current approver appears to be a smart contract or not. Fine to do as a follow-up PR

Reply:
kk -- i will do as follow up.  i have another pr in progress adding reserve cli tooling, so i'll include this